speech/snippets/quickstart_v2.py

```diff
@@ -21,33 +21,24 @@
 
 def quickstart_v2(
     project_id: str,
-    recognizer_id: str,
     audio_file: str,
 ) -> cloud_speech.RecognizeResponse:
     """Transcribe an audio file."""
     # Instantiates a client
     client = SpeechClient()
 
-    request = cloud_speech.CreateRecognizerRequest(
-        parent=f"projects/{project_id}/locations/global",
-        recognizer_id=recognizer_id,
-        recognizer=cloud_speech.Recognizer(
-            language_codes=["en-US"], model="latest_long"
-        ),
-    )
-
-    # Creates a Recognizer
-    operation = client.create_recognizer(request=request)
-    recognizer = operation.result()
-
     # Reads a file as bytes
     with open(audio_file, "rb") as f:
         content = f.read()
 
-    config = cloud_speech.RecognitionConfig(auto_decoding_config={})
+    config = cloud_speech.RecognitionConfig(
+        auto_decoding_config={}, language_codes=["en-US"], model="latest_long"
+    )
 
     request = cloud_speech.RecognizeRequest(
-        recognizer=recognizer.name, config=config, content=content
+        recognizer=f"projects/{project_id}/locations/global/recognizers/_",
```

Comment:
Is this underscore at the end intentional?

Reply:
Yes, this is a new feature 🎉 you can now make recognition requests without creating a recognizer, this `_` is an "empty" recognizer. I'll update the appropriate code samples as I go (plus add new ones)